docs/build/implementations.md

```diff
@@ -29,11 +29,11 @@ Here is a snapshot of each implementation’s progress across the primary parts
 
 ✅ : fully featured implementation | 🔄 : reuses components from another implementation | 🔶 : partial implementation
```

Comment:
1 icon per line would be easier to read:

✅ : fully featured implementation
🔄 : reuses components from another implementation
🔶 : partial implementation

Reply:
Good point. Let's see if I can add that without playing around with GitHub's funky editor.

```suggestion
✅ : fully featured implementation
🔄 : reuses components from another implementation
🔶 : partial implementation
```